apps/webpack.js

```diff
@@ -90,8 +97,71 @@ var karmaConfig = _.extend({}, config, {
   ]
 });
 
+/**
+ * Generate the appropriate webpack config based off of our base config and
+ * some input options
+ * @param {object} options
+ * @param {string} options.output
+ * @param {string[]} options.entries - list of input source files
+ * @param {bool} options.minify
+ * @param {bool} options.watch
+ * @param {string} options.piskelDevMode
+ */
+function create(options) {
+  var outputDir = options.output;
```

Comment:
Actually, I think I'd like to leave it as is for the following reason - outputDir is also used elsewhere in Gruntfile.js. By keeping it defined there and passed into webpack.js, we can have it defined in only one spot.